packages/browser/src/transports/xhr.ts

```diff
@@ -36,29 +38,38 @@ export class XHRTransport extends BaseTransport {
       });
     }
 
-    return this._buffer.add(
-      () =>
-        new SyncPromise<Response>((resolve, reject) => {
-          const request = new XMLHttpRequest();
+    return this._buffer
+      .add(
+        () =>
+          new SyncPromise<Response>((resolve, reject) => {
+            const request = new XMLHttpRequest();
 
-          request.onreadystatechange = (): void => {
-            if (request.readyState === 4) {
-              const headers = {
-                'x-sentry-rate-limits': request.getResponseHeader('X-Sentry-Rate-Limits'),
-                'retry-after': request.getResponseHeader('Retry-After'),
-              };
-              this._handleResponse({ requestType: sentryRequest.type, response: request, headers, resolve, reject });
-            }
-          };
+            request.onreadystatechange = (): void => {
+              if (request.readyState === 4) {
```

Comment:
Not related to this PR, but we may want to have an enum (or another structure) to replace this `4`.

Reply:
Not sure why doesnt TS types do that as well